app/views/browse/config-maps.html

```diff
@@ -26,38 +26,46 @@
       <alerts alerts="alerts"></alerts>
       <div class="row">
         <div class="col-md-12">
-          <div ng-if="!loaded">Loading...</div>
-          <div ng-if="loaded">
-            <table class="table table-bordered table-mobile table-layout-fixed">
-              <colgroup>
-                <col class="col-sm-5">
-              </colgroup>
-              <thead>
-                <tr>
-                  <th>Name</th>
-                  <th>Created</th>
-                  <th>Labels</th>
-                </tr>
-              </thead>
-              <tbody ng-if="(configMaps | hashSize) == 0">
-                <tr><td colspan="3"><em>No config maps to show</em></td></tr>
-              </tbody>
-              <tbody ng-if="(configMaps | hashSize) > 0">
-                <tr ng-repeat="configMap in configMaps">
-                  <td data-title="Name">
-                    <a href="{{configMap | navigateResourceURL}}">{{configMap.metadata.name}}</a>
-                  </td>
-                  <td data-title="Created">
-                    <span am-time-ago="configMap.metadata.creationTimestamp"></span>
-                  </td>
-                  <td data-title="Labels">
-                    <em ng-if="(configMap.metadata.labels | hashSize) === 0">none</em>
-                    <labels labels="configMap.metadata.labels" clickable="true" kind="Config Map" project-name="{{configMap.metadata.namespace}}" limit="3" filter-current-page="true"></labels>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+          <div ng-if="(configMaps | hashSize) == 0">
```

Comment:
Hi, would be great to use the [pfEmptyState](https://www.patternfly.org/angular-patternfly/#/api/patternfly.views.component:pfEmptyState) component.

Reply:
@dtaylor113, I opted to use the pre-existing empty-state styles that pre-date pfEmptyState that were already in use in the console for the sake of consistency.